src/middlewared/middlewared/alert/source/applications.py

```diff
@@ -1,6 +1,21 @@
 from middlewared.alert.base import AlertClass, AlertCategory, AlertLevel, Alert, OneShotAlertClass
 
 
+class FailuresInAppMigrationAlertClass(AlertClass, OneShotAlertClass):
+    deleted_automatically = False
+
+    category = AlertCategory.APPLICATIONS
+    level = AlertLevel.ERROR
+    title = 'Application(s) failed to migrate from kubernetes'
```

Comment:
Don't think we need to list kubernetes here either. It's an implementation detail.